testing/tests/DevExpress.ui.widgets.dataGrid/columnsHeadersView.tests.js

```diff
@@ -792,18 +792,18 @@ QUnit.test('Apply sorting by click', function(assert) {
     // arrange
     var testElement = $('#container');
 
-    $.extend(this.columns, [{ index: 0, alignment: 'right', allowSorting: true }]);
+    $.extend(this.columns, [{ index: 0, alignment: 'right', allowSorting: true }, { index: 1, dataField: "isDirectory", caption: 'Column 2', allowSorting: true }]);
```

Comment:
You have written tests in data grid fixture. The tests test data grid functionality, but we should test file manager features.

Please, create new test that create file manager and test sorting of data grid among file manager enviroment.

Reply:
Tests are moved to file_manager/detailsView.tests 
Check it, please.